pages/events.js

```diff
@@ -30,7 +29,54 @@ export default function Events({ events }) {
       event.date.cfpClose ? new Date(event.date.cfpClose) > new Date() : false
     ),
   };
+  const filters = [
+    {
+      title: "Show all",
+      description: "List all events with no filters",
+      key: "all",
+      icon: FaListUl,
+      current: true,
+    },
+    {
+      title: "CFP open",
+      description: "You can submit a talk to this conference",
+      key: "cfpOpen",
+      icon: FaMicrophoneAlt,
+      current: false,
+    },
+    {
+      title: "In person",
+      description: "These are in person events",
+      key: "inPerson",
+      icon: MdOutlinePeople,
```

Comment:
I noticed these icons aren't used in the `EventTabs.js` the icons are hard coded in the file. Could you tell me where these are used?

Reply:
These were being used in `EventKey.js` to create these cards 
![image](https://user-images.githubusercontent.com/95094057/211858177-86f9d72d-6d94-4a8d-9af7-6ad21e81c102.png)